docs/resources/v2-conformance-statement.md

```diff
@@ -0,0 +1,818 @@
+- Paged results are optimized to return matched *newest* instance first, this may result in duplicate records in subsequent pages if newer data matching the query was added.
+- Matching is case in-sensitive and accent in-sensitive for PN VR types.
+- Matching is case in-sensitive and accent sensitive for other string VR types.
+- Only the first value will be indexed of a single valued data element that incorrectly has multiple values.
+- For best performance default attributes or limited number of responses is recommended.
+
+## Delete
+
+This transaction is not part of the official DICOMweb&trade; Standard. It uses the DELETE method to remove representations of Studies, Series, and Instances from the store.
+
+| Method | Path                                                    | Description |
+| :----- | :------------------------------------------------------ | :---------- |
+| DELETE | ../studies/{study}                                      | Delete all instances for a specific study. |
+| DELETE | ../studies/{study}/series/{series}                      | Delete all instances for a specific series within a study. |
+| DELETE | ../studies/{study}/series/{series}/instances/{instance} | Delete a specific instance within a series. |
+
+Parameters `study`, `series` and `instance` correspond to the DICOM attributes StudyInstanceUID, SeriesInstanceUID and SopInstanceUID respectively.
+
+There are no restrictions on the request's `Accept` header, `Content-Type` header or body content.
+
+> Note: After a Delete transaction the deleted instances will not be recoverable.
+
+### Response Status Codes
+
+| Code                         | Description |
+| :--------------------------- | :---------- |
+| 204 (No Content)             | When all the SOP instances have been deleted. |
+| 400 (Bad Request)            | The request was badly formatted. |
+| 401 (Unauthorized)           | The client is not authenticated. |
+| 401 (Unauthorized)           | The client isn't authenticated. |
+| 403 (Forbidden)              | The user isn't authorized. |
+| 404 (Not Found)              | When the specified series was not found within a study, or the specified instance was not found within the series. |
+| 503 (Service Unavailable)    | The service is unavailable or busy. Please try again later. |
+
+### Delete Response Payload
+
+The response body will be empty. The status code is the only useful information returned.
+
+# Worklist Service (UPS-RS)
+
+The DICOM service supports the Push and Pull SOPs of the [Worklist Service (UPS-RS)](https://dicom.nema.org/medical/dicom/current/output/html/part18.html#chapter_11). This service provides access to one Worklist containing Workitems, each of which represents a Unified Procedure Step (UPS).
+
+Throughout, the variable `{workitem}` in a URI template stands for a Workitem UID.
+
```

Comment:
There is an entire table missing here - Available UPS-RS endpoints.  See https://learn.microsoft.com/en-us/azure/healthcare-apis/dicom/dicom-services-conformance-statement#worklist-service-ups-rs

Reply:
hmm. I did not remove UPS-RS. Since I created a new file, it is hard to see the diff. Best way is to download the file and compare with your local conformance file.

I see the diff in only in default reduction and adding them to All response.

But your assumption is right, we are reducing the default to make it more performant.